typescript/web/next-sitemap.js

```diff
@@ -0,0 +1,20 @@
+const { VERCEL_URL } = process.env;
+
+if (!VERCEL_URL) {
+  throw new Error("Missing required environment variable VERCEL_URL");
+}
+
+const SITE_URL = VERCEL_URL.includes("://")
+  ? VERCEL_URL
+  : `https://${VERCEL_URL}`;
+
+module.exports = {
+  siteUrl: SITE_URL,
+  exclude: [
+    "/settings/profile",
+    "/debug",
+    "/thank-you",
+    "/request-access",
+    "/graphiql",
```

Comment:
The exclude didn't work: https://labelflow-ix6ocxwtv-labelflow.vercel.app/sitemap.xml

Reply:
It's not the good preview, look at [the last one](https://labelflow-8am44mudc-labelflow.vercel.app/sitemap.xml)
They are gone 😉

Reply:
Oh sorry, I've missed the unique ID in the URL 🤦‍♂️ Then I'll approve your pr. Good job 👏